sufia-models/lib/sufia/models/utils.rb

```diff
@@ -14,6 +28,7 @@ def retry_unless(number_of_tries, condition, &block)
         number_of_tries.each do
           result = block.call
           return result unless condition.call
+          sleep(Sufia.config.retry_unless_sleep) if Sufia.config.retry_unless_sleep > 0
```

Comment:
What value would that be set to other than a non-zero value?  Zero, the default?  If so, doesn't sleeping for zero preserve backward compatibility?